cmd/cli/plugin/secret/registry_secret_add.go

```diff
@@ -74,16 +84,40 @@ func registrySecretAdd(cmd *cobra.Command, args []string) error {
 			}
 		}
 		log.Info("\n")
-	}
-
-	pkgClient, err := tkgpackageclient.NewTKGPackageClient(registrySecretOp.KubeConfig)
-	if err != nil {
-		return err
-	}
+	} else {
+		secretExportList, err := pkgClient.ListSecretExports(registrySecretOp)
+		if err != nil {
+			return err
+		}
 
-	kc, err := kappclient.NewKappClient(registrySecretOp.KubeConfig)
-	if err != nil {
-		return err
+		for j := range secretExportList.Items {
+			export := ""
+			secretExport := secretExportList.Items[j]
+			if secretExport.Name == registrySecretOp.SecretName {
+				ns := &corev1.NamespaceList{}
+
+				err = kc.GetClient().List(context.Background(), ns)
+				if err != nil {
+					return err
+				}
+
+				if findInList(secretExport.Spec.ToNamespaces, "*") || secretExport.Spec.ToNamespace == "*" {
+					export = "all namespaces"
+				} else {
+					export = "some namespaces"
+				}
+
+				// Ask user consent when SecretExport has been created by kubectl and user tries to add secret of the same name as SecretExport without using --export-to-all-namespaces flag
+				log.Warningf("Warning: SecretExport with the same name exists already, given secret contents will be available to %s. If you decide not to proceed, you can either delete the SecretExport or specify a different secret name.\n\n", export)
```

Comment:
nit-pick: we might not want to add `Warning:` since `log.Warningf` should already have that prepended.

Reply:
As far as I can tell, ```log.Warningf``` doesn't prepend a "Warning"

Reply:
Output is:
"Warning: SecretExport with the same name exists already, given secret contents will be available to all namespaces. If you decide not to proceed, you can either delete the SecretExport or specify a different secret name."